### PR TITLE
Use a different flag parser for “nicer” flags

### PR DESCRIPTION
**Pull request by matiaskorhonen**
Switches to the [Kingpin](https://github.com/alecthomas/kingpin) flag parser so that we can have both long (`--help`) and short ( `-h`) flags.

``` sh
$ goad --help
usage: goad --url=URL [<flags>]

A command-line chat application.

Flags:
  -h, --help                     Show context-sensitive help (also try --help-long and --help-man).
  -u, --url=URL                  URL to load test
  -m, --method="GET"             HTTP method
  -b, --body=BODY                HTTP request body
  -c, --concurrency=10           Number of concurrent requests
  -n, --requests=10              Total number of requests to make
  -t, --timeout=15               Request timeout in seconds
  -r, --regions="us-east-1,eu-west-1,ap-northeast-1"
                                 AWS regions to run in (comma separated, no spaces)
  -p, --aws-profile=AWS-PROFILE  AWS named profile to use
  -H, --headers=HEADERS ...      List of headers
      --version                  Show application version.

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,22 +62,24 @@ Goad will read your credentials from `~/.aws/credentials` or from the `AWS_ACCES
...
-  -t uint
-      request timeout in seconds (default 15)
-  -u string
-      URL to load test (required)
-  -H string
-      HTTP Header to add to the request. This option can be set multiple times.
+$ goad --help
+usage: goad --url=URL [<flags>]
+
+A command-line chat application.
```
Comment (zeph) [inline README.md]: ```A command-line chat application.```
??? copy paste? 

besides... I like the new structure, but I wonder how many people have already scripted the goad usage based on the old params (which is basically like an API/interface to other tools) ...I'm not a maintainer of goad, but I doubt this patch might be accepted

Comment (matiaskorhonen) [inline README.md]: I fixed the copy-pasta.

The idea is to keep both the current short flags and introduce clearer, more verbose long flags for clarity.

**Comment by gserra-olx** [comment]
@matiaskorhonen the CI is failing on this pull request, would u mind checking ur changes?

**Comment by zeph** [comment]
closing, code has already been rewritten related to the CLI interface

**Pull request closed without merging**